lstchain/scripts/longterm_dl1_check.py

```diff
@@ -1,47 +1,84 @@
 #!/usr/bin/env python
 
 """
-This script has to be run in a directory containing LST1 DL1 datacheck files, of
-those containing inf for a whole run, with name pattern
-datacheck_dl1_LST-1.Run?????.h5
-
-The corresponding muon*fits files (which are subrun-wise) have to be present
-in the same directory.
-
-The output is the file longterm_dl1_check.h5 file, which contains tables with
-some run-wise summary values for plotting long-term evolution of the DL1 data.
+This script reads in LST1 DL1 datacheck files, of those containing info for a
+whole run, with name pattern datacheck_dl1_LST-1.Run?????.h5. It takes all
+files in the input directory. It also reads, if available in the same path,
+the corresponding muons*fits files.
+
+The output is the file longterm_dl1_check.h5 file (the name can be modified via
+commandline), which contains tables with some run-wise summary values for
+plotting long-term evolution of the DL1 data.
 It also produces an interactive web page, longterm_dl1_check.html with plots
-showing the evolution of many such values.
+showing the evolution of many such values. If not in batch mode, the page is
+opened by the default browser at the end of execution.
+
+It also produces a longterm_dl1_check.log file with warnings about values
+which are beyond certain limits.
 
 """
 
+import argparse
+import logging
 import copy
-import glob
 from pathlib import Path
 
 import numpy as np
 import pandas as pd
 import tables
 from astropy.table import Table
 from bokeh.io import output_file as bokeh_output_file
-from bokeh.io import show
+from bokeh.io import show, save
 from bokeh.layouts import gridplot, column
 from bokeh.models import Div, ColumnDataSource, Whisker, HoverTool, Range1d
 from bokeh.models.widgets import Tabs, Panel
 from bokeh.plotting import figure
 from ctapipe.coordinates import EngineeringCameraFrame
 from ctapipe.instrument import SubarrayDescription
+from ctapipe_io_lst import TriggerBits
+
+from lstchain.visualization.bokeh import show_camera, get_pixel_location
+
+log = logging.getLogger(__name__)
+
+parser = argparse.ArgumentParser(description="DL1 multi-run data checker")
+parser.add_argument('--input-dir', '-d', action='store', type=Path,
+                    dest='input_dir',
+                    help='path to the datacheck_dl1_LST-1.RunXXXXX.h5 files',
+                    default='./')
```

Comment:
Currently, muons files are not placed in the same directory as the datacheck*h5 files, can we add a muons-dir extra argument?